charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,80 @@
+
+# Project Name
+HCA Data Processing Service
+
+## Description
+
+The data processing service is a centralized space to perform all transformative functionality to scientific data ingested into the Human Cell Atlas Data Coordination Platform (HCA DCP).
+
+## Definitions
+
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
+__HCA Analysis Working Group (AWG):__ A group of domain specialists that advise on and monitor scientific choices made in the Human Cell Atlas Project.  
+__HCA DCP Governance Group:__ A working group of the Human Cell Atlas project that advises the DCP, assuring alignment of DCP development efforts with the scientific project it supports.  
+__On-boarding:__ The process of taking an existing pipeline and engineering it to work in the HCA DCP.  
+__Scoping:__ The process of investigating the available resources and knowledge in a community around a specific assay to better understand how much work it would take to make a pipeline to process data from that assay.  
+__Standardized Pipeline:__ An official pipeline of the HCA project.  
+
+## Objectives
+
+The data processing service aims to develop scientific analysis pipelines and the infrastructure to run them reliably and at-scale for all analyses on all scientific data types in the Human Cell Atlas Data Coordination Platform. The team working on the service will also operate the infrastructure. Pipelines for additional data types, analyses, and improvements to these pipelines will be added iteratively and in collaboration with the HCA Analysis Working Group (AWG) and the community.
+
+## In-scope
+
+* Scoping or review of scoping for new pipelines.  
```

Comment:
* Like above ⬆️, remove periods from non-sentences in a list.
* Preferably, write all bullets in active voice: `Create, maintain, and operate ...`. Apply this consistently throughout the doc.

Reply:
thank you for the feedback.